src/Sylius/Behat/Context/Ui/CheckoutContext.php

```diff
@@ -96,6 +97,29 @@ public function iProceedSelectingOfflinePaymentMethod($paymentMethodName)
     }
 
     /**
+     * @When /^I proceed selecting "([^"]*)" shipping method$/
+     */
+    public function iProceedSelectingShippingMethod($shippingMethod)
+    {
+        $checkoutAddressingPage = $this->getPage('Checkout\CheckoutAddressingStep')->open();
+        $addressingDetails = array(
+            'firstName' => 'John',
+            'lastName' => 'Doe',
+            'country' => 'France',
+            'street' => '0635 Myron Hollow Apt. 711',
+            'city' => 'North Bridget',
+            'postcode' => '93-554',
+            'phoneNumber' => '321123456'
+        );
+        $checkoutAddressingPage->fillAddressingDetails($addressingDetails);
+        $checkoutAddressingPage->pressButton('Continue');
+
+        $checkoutShippingPage = $this->getPage('Checkout\CheckoutShippingStep');
+        $checkoutShippingPage->pressRadio($shippingMethod);
```

Comment:
Can we extract it to `$checkoutShippingPage->selectShippingMethod($shippingMethod)`?